continuous_integration/install.sh

```diff
@@ -81,5 +81,10 @@ else
     exit 1
 fi
 
-pip install coverage coveralls
+pip install coverage
```

Comment:
This should go into the "if" condition. If we decide to run coverage on only one of the runs, then we should skip it in the others as it requires a significant time to run.